tests/FSharp.Compiler.UnitTests/ByteMemoryTests.fs

```diff
@@ -3,14 +3,15 @@ namespace FSharp.Compiler.UnitTests
 
```

Comment:
Should the test files be all in the same directory ````ByteMemoryTests.fs, EditDistance.fs, HashIfExpression.fs.```` or should they be further differentiated, it's just that I imagine eventually Compiler.ComponentTest would have a huge number of files in it.

Reply:
@KevinRansom my idea was grouping files by the directories inside the test suites (e.g. Compiler.ComponentTests). But they can as well be factored out into separate suite if will grow in size.